schedule/yast/sle_15-sp4/install/usb_install.yaml

```diff
@@ -0,0 +1,20 @@
+name: USBinstall_Online
+description: >
+  Basic installation test with USB boot. Verification that usb repo is enabled
+  after first boot. Test cover Online medium installations.
+vars:
+  DESKTOP: textmode
+  USBBOOT: 1
+  YUI_REST_API: 1
+schedule:
+  default_systemd_target:
+    - installation/installation_settings/validate_default_target
+  grub:
+    - installation/grub_test
```

Comment:
not needed, `handle_reboot` will do the job, but we should find in the future some way to make it more readable, as there is not reconnection or any remote in this case for example, but afir there was some controversy with the name in PR creating that test module and got that `handle_reboot` instead of `handle_grub`, I would say let's use the default here for now.